conftest.py

```diff
@@ -0,0 +1,8 @@
+def pytest_configure():
+    from django.conf import settings
+    settings.configure(
+        URLS_EXCLUDED_FROM_SIGNATURE_CHECK=[],
+        SIGNATURE_AUTH_INBOUND_SECRET='really-secret-host',
```

Comment:
where are `SIGNATURE_AUTH_INBOUND_SECRET` and `SIGNATURE_AUTH_OUTBOUND_SECRET` used?